src/WorkOS.net/Services/SSO/_interfaces/GetAuthorizationURLOptions.cs

```diff
@@ -52,12 +52,10 @@ public class GetAuthorizationURLOptions : BaseOptions
         public string Organization { get; set; }
 
         /// <summary>
-        /// An optional parameter that specifies the type of Connection to
-        /// authenticate with. If used, a user of any domain can be
-        /// authenticated. Only `GoogleOAuth` is currently supported.
+        /// Used to initiate authentication with OAuth.
         /// </summary>
         [JsonProperty("provider")]
-        public ConnectionType? Provider { get; set; }
```

Comment:
This was a legacy input param. Since we are cutting a major version of the API anyway, we should update this to what the API is publicly documented to support.